queries.sql

```diff
@@ -1,15 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * FROM animals WHERE name LIKE '%mon';
-/* List the name of all animals born between 2016 and 2019*/
 SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
-/* Name of all animals that are neutered and have less than 3 escape attempts */
 SELECT * FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
-/* List the date of birth of all animals named either "Agumon" or "Pikachu"*/
 SELECT * FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
-/*List name and escape attempts of animals that weigh more than 10.5kg*/
 SELECT name FROM animals WHERE weight_kg > 10;
-/*Find all animals that are neutered*/
-SELECT name FROM animals WHERE neutered = TRUE;
-/* Find all animals not named Gabumon*/
-SELECT name FROM animals WHERE name NOT IN ('Gabumon');
\ No newline at end of file
+SELECT * FROM animals WHERE neutered = TRUE;
+SELECT * FROM animals WHERE name NOT IN ('Gabumon');
+SELECT * FROM animals WHERE weight_kg >=10.4 AND weight_kg <= 17.3;
```

Comment:
- [ ] Good job so far 👍 Kindly run all your queries and provide screenshots of the results in your PR description. Since there are 8 queries in total, you should have 8 screenshots in your PR description which show the results of the queries 👍